index.html

```diff
@@ -66,47 +69,49 @@
     <div class="row">
       <div class="col-md-3">
         <img class="img-responsive center-block bottom-pad" width="100" height="100" src="./assets/img/provenance.svg">
-        <p class="text-justify">Automatically track your analyses with
-        decentralized data provenance &mdash; no more guesswork on what commands were
-        run!</p>
+        <p class="text-justify">Integrated provenance tracking ensures bioinformatics reproducibility
+        and even allows you to <a href="https://forum.qiime2.org/t/provenance-replay-alpha-release-and-tutorial/23279" target="_blank">"replay"</a>
+        your analysis steps on the same or new data &mdash; no more guesswork on what commands were run!</p>
       </div>
       <div class="col-md-3">
         <img class="img-responsive center-block bottom-pad" width="100" height="100" src="./assets/img/viz.svg">
-        <p class="text-justify">Interactively explore your data with beautiful
-        visualizations that provide new perspectives.</p>
+        <p class="text-justify">Beautiful, interactive visualizations provide new perspectives and help translate your data into knowledge.</p>
       </div>
       <div class="col-md-3">
         <img class="img-responsive center-block bottom-pad" width="100" height="100" src="./assets/img/sharing.svg">
-        <p class="text-justify">Easily share results with your team, even those
-        members without QIIME 2 installed.</p>
+        <p class="text-justify">Easily <a href="https://view.qiime2.org">share your results</a> with your team or the world!
+        Viewers don't need to have QIIME 2 installed.</p>
       </div>
       <div class="col-md-3">
         <img class="img-responsive center-block bottom-pad" width="100" height="100" src="./assets/img/plugin.svg">
-        <p class="text-justify">Plugin-based system &mdash; your favorite microbiome methods all in one place.</p>
+        <p class="text-justify">QIIME 2 plugins put your favorite microbiome methods all in one place, and enable bioinformatics
+          developers to rapidly <a href="https://library.qiime2.org">share new methods</a> with QIIME 2 users.</p>
       </div>
     </div>
 
     <hr>
 
     <div>
-      <h3 id="interfaces" class="trim-the-top-margin">Choose the interface that fits your needs</h3>
+      <h3 id="interfaces" class="trim-the-top-margin">
+        QIIME 2's different interfaces allow you to work in the environment that you're most productive with.
+      </h3>
 
       <div class="row">
         <div class="col-md-4">
           <div class="row bottom-pad">
             <div class="col-md-12">
               <h4>
-                <a href="https://docs.qiime2.org/2022.11/interfaces/q2cli/">q2cli</a>
-                <small>the command line interface</small>
+                <a href="https://docs.qiime2.org/2022.11/interfaces/q2cli/">command line interface:</a>
```

Comment:
```suggestion
                <a href="https://docs.qiime2.org/2022.11/interfaces/q2cli/">command line interface</a>
```
I'd remove this colon from the title (or add colons to the other two interface titles) - just for consistency's sake

Reply:
will do, thanks!